platform/admin/core/admin-core-appservice/src/main/java/org/codice/ddf/admin/application/service/impl/ApplicationServiceImpl.java

```diff
@@ -161,9 +167,12 @@ public Application getApplication(String applicationName) {
 
     @Override
     public boolean isApplicationStarted(Application application) {
-        ApplicationStatus status = getApplicationStatus(application);
-        return (status.getState()
-                .equals(ApplicationState.ACTIVE));
+        for (Feature feature : application.getAutoInstallFeatures()) {
```

Comment:
Could replace this with...
```
return application.getAutoInstallFeatures().stream()
    .allMatch(featuresService::isInstalled);
```
(That method reference should work; if it doesn't, changing that to a lambda would.)